sites/tvtv.us/tvtv.us_us.channels.xml

```diff
@@ -623,54 +541,38 @@
-    <channel lang="en" xmltv_id="StingrayClassica.ca" site_id="67641">Stingray Classica</channel>
-    <channel lang="en" xmltv_id="StingrayClassicRock.ca" site_id="67567">Stingray Classic Rock</channel>
-    <channel lang="en" xmltv_id="StingrayEasyListening.ca" site_id="67573">Stingray Easy Listening</channel>
-    <channel lang="en" xmltv_id="StingrayTodaysLatinPop.ca" site_id="68639">Stingray Exitos del Momento</channel>
-    <channel lang="en" xmltv_id="StingrayFlashback70s.ca" site_id="67565">Stingray Flashback 70s</channel>
-    <channel lang="en" xmltv_id="StingrayGreatestHits.ca" site_id="110433">Stingray Greatest Hits</channel>
-    <channel lang="en" xmltv_id="StingrayUrbanBeat.ca" site_id="67564">Stingray Hip-Hop/R&amp;B</channel>
-    <channel lang="en" xmltv_id="StingrayHitList.ca" site_id="67555">Stingray Hit List</channel>
-    <channel lang="en" xmltv_id="StingrayHotCountry.ca" site_id="67557">Stingray Hot Country</channel>
-    <channel lang="en" xmltv_id="StingrayKaraoke.ca" site_id="110657">Stingray Karaoke</channel>
-    <channel lang="en" xmltv_id="StingrayNaturescape.us" site_id="90613">Stingray Naturescape</channel>
-    <channel lang="en" xmltv_id="StingrayNothinBut90s.ca" site_id="68620">Stingray Nothin' But 90's</channel>
-    <channel lang="en" xmltv_id="StingrayPopAdult.ca" site_id="67558">Stingray Pop Adult</channel>
-    <channel lang="en" xmltv_id="StingrayEverything80s.ca" site_id="67566">Stingray Remember the 80's</channel>
-    <channel lang="en" xmltv_id="StingrayRockAlternative.ca" site_id="67568">Stingray Rock Alternative</channel>
-    <channel lang="en" xmltv_id="StingraySmoothJazz.ca" site_id="67571">Stingray Smooth Jazz</channel>
-    <channel lang="en" xmltv_id="StingraySoulStorm.ca" site_id="67563">Stingray Soul Storm</channel>
     <channel lang="en" xmltv_id="Studiocanal.fr" site_id="35751">StudioCanal</channel>
     <channel lang="en" xmltv_id="SundanceTVEast.us" site_id="71280">SundanceTV East</channel>
     <channel lang="en" xmltv_id="SundanceTVWest.us" site_id="78806">SundanceTV West</channel>
     <channel lang="en" xmltv_id="SyfyEast.us" site_id="58623">Syfy East</channel>
     <channel lang="en" xmltv_id="SyfyWest.us" site_id="65626">Syfy West</channel>
-    <channel lang="en" xmltv_id="Tastemade.us" site_id="107076">Tastemade</channel>
+    <channel lang="en" xmltv_id="Tastemade.us" site_id="107076">Tastemade (Streaming)</channel>
     <channel lang="en" xmltv_id="TBNEast.us" site_id="14767">TBN East</channel>
     <channel lang="en" xmltv_id="TBNInspire.us" site_id="122276">TBN Inspire</channel>
     <channel lang="en" xmltv_id="TBSEast.us" site_id="58515">TBS East</channel>
     <channel lang="en" xmltv_id="TBSWest.us" site_id="67890">TBS West</channel>
-    <channel lang="en" xmltv_id="TCMEast.us" site_id="12852">TCM</channel>
+    <channel lang="en" xmltv_id="TCMEast.us" site_id="12852">Turner Channel Movies East</channel>

```

Comment:
TCMEast.us is probably Turner Classic Movies East. (https://en.wikipedia.org/wiki/Turner_Classic_Movies)